src/internal_events/prometheus.rs

```diff
@@ -129,19 +131,13 @@ impl InternalEvent for PrometheusRemoteWriteSnapError {
 
 #[derive(Debug)]
 pub struct PrometheusRemoteWriteReceived {
-    pub byte_size: usize,
```

Comment:
Why is this removed?

Reply:
Because the `emit_metrics` that was using it was removed.